### PR TITLE
Database and Server - WIP 

### PR DESCRIPTION
**Pull request by kiannaquach**
This pull request is to explore my implementation of seeding mock data to my database as well as my api call to the server. Looking for people to review my code.

### CODE REVIEW

File: client/components/ProductDetails.jsx [hunk 1]
```diff
@@ -9,11 +9,10 @@ class ProductDetails extends React.Component {
       products: []
     }
   }
-
   componentDidMount() {
     axios.get('/productDetails')
     .then((response) => {
```
Comment (lhbrennan) [inline client/components/ProductDetails.jsx]: the airbnb guide requires that these thens get indented. i'm fine with the way you have it, but let's chat with the group so all agree on a standard.

File: client/components/ProductDetails.jsx [hunk 2]
```diff
@@ -25,19 +24,28 @@ class ProductDetails extends React.Component {
 
   render() {
     return(
-        <div>
-          <h1>[******Product Details Component Rendered******]</h1>
+      <div>
+          <h1>lululemonades product details</h1>
+      <div className="product-details">
```
Comment (lhbrennan) [inline client/components/ProductDetails.jsx]: looks like this one should be indented more

File: databases/index.js [hunk 3]
```diff
@@ -20,38 +20,31 @@ let productSchema = mongoose.Schema({
...
-
-            products.save()
-                .then((data) => {
-                    console.log('YOU INSERT TO DB', data)
-                }).catch((err) => {
-                    console.log('YOU FAILED INSERT', err)
-                });
-        })()
-    }
+// iterate through item
```
Comment (lhbrennan) [inline databases/index.js]: can safely be removed now -- i think the next block of code is self-explanatory

File: databases/index.js [hunk 4]
```diff
@@ -20,38 +20,31 @@ let productSchema = mongoose.Schema({
...
+            "color": itemDetails[i].color,
+            "size": itemDetails[i].size,
+            "id": itemDetails[i].id
+        });
+    })()
 }
 
-
-ProductDetail.find()
+products.save()
```
Comment (lhbrennan) [inline databases/index.js]: I'm confused -- seems like this block of code needs to be inside of the previous for loop

File: databases/index.js [hunk 5]
```diff
@@ -20,38 +20,31 @@ let productSchema = mongoose.Schema({
...
-    })
+        // console.log('YOU FAILED INSERT', err)
+    });
+
 
 
-module.exports.retrieve = retrieve;
\ No newline at end of file
+module.exports = {
+    retrieve,
```
Comment (lhbrennan) [inline databases/index.js]: looks like 4 space indentations here

File: server/server.js [hunk 6]
```diff
@@ -14,11 +14,14 @@ app.listen(port, () => {
 });
 
 app.get('/productDetails', (req, res) => {
-    db.retrieve()
```
Comment (lhbrennan) [inline server/server.js:17]: looks like 4 space indentation

File: server/server.js [hunk 7]
```diff
@@ -14,11 +14,14 @@ app.listen(port, () => {
...
+    db.ProductDetail.find()
         .then((data) => {
-            res.send(data);
+            // console.log('YOUR DATA', data);
+            res.send(data)
+            res.end()
```
Comment (lhbrennan) [inline server/server.js]: I believe res.send() also ends it, so the res.end() may be redundant

**Review by lhbrennan — COMMENTED** [review]
overall really good! easy to follow along with.

File: server/server.js (hunk 6 above)
Comment (dorktobes) [reply server/server.js:17]: Probably don't need to worry about style on removed lines :-P

**Pull request merged**